pkg/strategy/bollmaker/strategy.go

```diff
@@ -515,18 +494,28 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 	s.orderExecutor.BindEnvironment(s.Environment)
 	s.orderExecutor.BindProfitStats(s.ProfitStats)
 	s.orderExecutor.Bind()
-
 	s.orderExecutor.TradeCollector().OnPositionUpdate(func(position *types.Position) {
 		bbgo.Sync(s)
 	})
-
-	s.SmartStops.RunStopControllers(ctx, session, s.orderExecutor.TradeCollector())
+	s.ExitMethods.Bind(session, s.orderExecutor)
 
 	if bbgo.IsBackTesting {
 		log.Warn("turning of useTickerPrice option in the back-testing environment...")
 		s.UseTickerPrice = false
 	}
 
+	s.OnSuspend(func() {
+		s.Status = types.StrategyStatusStopped
```

Comment:
This line is not necessary. Strategy controller changes it for you

Reply:
updated